lib/fcrepo_wrapper/instance.rb

```diff
@@ -150,20 +150,22 @@ def options
     end
 
     ##
-    # Clean up any files fcrepo_wrapper may have downloaded
-    def clean!
+    # Clean up any files fcrepo_wrapper manages
+    def clean!(clean_downloads = false)
       stop
       remove_instance_dir!
-      FileUtils.remove_entry(config.download_path) if File.exists?(config.download_path)
-      FileUtils.remove_entry(config.tmp_save_dir, true) if File.exists? config.tmp_save_dir
-      md5.clean!
-      FileUtils.remove_entry(config.version_file) if File.exists? config.version_file
+      FileUtils.remove_entry(config.tmp_save_dir, true)
+      FileUtils.remove_file(config.version_file, true)
+      if clean_downloads
+        md5.clean!
+        FileUtils.remove_file(config.download_path, true)
+      end
     end
 
     ##
     # Clean up any files in the fcrepo instance dir
     def remove_instance_dir!
-      FileUtils.remove_entry(config.instance_dir, true) if File.exists? config.instance_dir
+      FileUtils.remove_entry(config.instance_dir, true)
     end
```

Comment:
Note, when `FileUtils.remove_*` is called with the `force: true`, it ignores any StdErr on failures.  Hence, the `File.exists?` checks are removed here.